flottform/forms/src/internal.ts

```diff
@@ -104,7 +106,7 @@ export function setIncludes<T>(set: Set<T>, x: T): boolean {
 	return false;
 }
 
-type Listener<T extends Array<any>> = (...args: T) => void;
+type Listener<T extends Array<any>> = (...args: T) => void; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Not sure how to improve this, to be honest. Wondering how we could improve this type

Reply:
I'd create a new issue for this so we can have a closer look later 👍🏼